doc/mgmt/gnmi/SONiC_GNMI_Server_Interface_Design.md

```diff
@@ -164,7 +164,12 @@ The ultimate goal is to use SONiC Yang models for configuration, and we also nee
 
 * We will follow OpenConfig specification to use origin field to support mixed schema.
 
-* And we use the first element to specify the target database.
+* We use the first element to specify the target database.
+
+* We use the second element to specify the database instance:
+  * For single asic device, this element is "localhost"
+  * For multiple asic device, this element can be "localhost", "asic0", "asic1", ...
+  * For smartswitch device, this element can be "localhost", "dpu0", "dpu1", ...
```

Comment:
Could you clarify that smartswitch device with multiple asic NPU is also possible in this design?